buzzmobile/launch/simulation.launch

```diff
@@ -9,7 +9,7 @@
         <include file="$(find buzzmobile)/launch/includes/mapping.launch"/>
         <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
         <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
-        <include file="$(find buzzmobile)/launch/includes/controller.launch"/>
+        <!--<include file="$(find buzzmobile)/launch/includes/controller.launch"/>-->
```

Comment:
but you still need car_pose_mux. That's the thing that takes in auto_car_pose and outputs car_pose.